app/shared/user-agent-client.js

```diff
@@ -17,10 +17,15 @@ import { logger } from './logging';
 import { UserAgentHttpClient } from './user-agent-http-client';
 
 class UserAgentClient extends EventEmitter {
-  constructor() {
+  // You'd think { initialConnectDelayMillis = 50, maxConnectDelayMillis = 10000 } would work.  In
+  // fact, you'd be crying in your beer, just like me: it doesn't when you call simply `new
+  // UserAgentClient()`.
+  constructor(options) {
```

Comment:
Thanks!  I will update with your suggestion, which I interpret to be ```
constructor({ x = 50, y = ... } = {})
```

As for productive, I disagree.  I commented specifically because it looks like it's not "the right thing", and it might save somebody pain who tries to make it look like "the right thing".  The test error this incites is that renderer tests hang with no message or exception, which is pretty hard to traceback to an innocent style change...